students/BRACH_Jakub/lesson_03_Functions/Calculator.py

```diff
@@ -0,0 +1,93 @@
+def SplashScreen():
+    print("{C} Boogie Soft inc. 2018 Welcome to badly organized calculator:")
+    ShowHelp()
+
+
+def RunAdd():
+    print("ADDING")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 + add_var_2)
```

Comment:
```
   print("Input 1st operand:") 
   var_1 = int(input())  
   print("Input 2nd operand:") 
   var_2 = int(input()) 
```
Getting input from user repeats in methods, it could be extracted to separate method.